### DISCUSSION TITLE
GitHub Actions is currently disabled

### DISCUSSION THREAD

**Discussion by cvatricks** [community/community#36309] [Actions]
Dear team,

Unfortunately I noticed in my account that "GitHub Actions is currently disabled for your account. Please reach out to [GitHub Support](https://support.github.com/contact?tags=dotcom-actions) for assistance."

Kindly please help me to retain GitHub Actions in my account.

Regards,
Cvatricks

**Comment by airtower-luna** [comment]
You'll have to do what the message says and [contact GitHub support](https://support.github.com/request) about that, us other users can't help there. Most likely some of your workflows triggered an abuse detection mechanism. Good luck!

**Comment by ossanna16** [comment]
Hey there @cvatricks and thank you for asking a good question! I'm glad you found our community 🙂
